tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
Exactly - I'm asking for a single way to set up credentials and optionally display them if we make a some change in the yml file.